build/autotools/CheckZlib.m4

```diff
@@ -12,6 +12,7 @@ AS_IF([test "x${with_zlib}" = xauto -o "x${with_zlib}" = xsystem], [
       AC_CHECK_LIB([zlib], [compress2], [
```

Comment:
Should "zlib" be uppercased here, too?: "AC_CHECK_LIB([ZLIB] ..."

Reply:
As far as I understand it, no. This argument is used to construct the library name, that is, `ZLIB` would result in `libZLIB`.

Reply:
No, it seems that "zlib" is correct here after all. Zlib installs a file at "${prefix}/lib/pkgconfig/zlib.pc", so pkg-config must search for "zlib", not "z".